tools/analyzer_plugin/README.md

```diff
@@ -0,0 +1,100 @@
+# OverReact Analyzer Plugin
+
+> A [Dart analyzer plugin][analyzer_plugin] for OverReact
+
+
+## Try it in your package!
+1. Add over_react to your pubspec.yaml
+
+    __If you wish to develop the plugin locally, see [this section](#local-development).__
+    
+1. Enable the plugin in your analysis_options.yaml:
+    ```yaml
+    analyzer:
+      plugins:
+        over_react
+    ```
+1. Restart the Dart Analysis Server in your IDE
+   
+## Repo Structure
+
+See the [analyzer_plugin package structure documentation][analyzer_plugin_package_structure] for terminology and more info.
+
+- _over_react_: the "host" package
+- _over_react_analyzer_plugin_ (`over_react/tools/analyzer_plugin`) - the "boostrap" and "plugin" packages, merged
+        
+    We decided to merge these packages since it allows us to avoid creating a separate package for the plugin, which would have resulted in a more painful dev/release experience. (Since the plugin depends on over_react, we'd want to use monorepo to manage the packages. However, we can't do that currently, due to internal tooling restrictions that prevent us from having multiple packages declared in a single repository.)
+        
+- _playground_ - (`over_react/tools/analyzer_plugin/playground`) - a "target" package that consumes the plugin, useful for manually testing plugin during development 
+        
+## Local Development
+
+See [analyzer_plugin's tutorial][analyzer_plugin_tutorial] on building a plugin for general information on developing a Dart analyzer plugin.
+
+### Setup
+
+**Before you do anything:** run 
+```sh
+dart tool/init_local_dev.dart
+```
+
+Normally, the Dart Analysis Server does a one-time copy of plugin code into the Dart Analysis Server state folder (usually `~/.dartServer/.plugin_manager`), and never updates for plugins derived from path dependencies (TODO make dart-lang issue for this). 
+
+This means, normally, ___any changes you make to the plugin will not be reflected___.
+
+This script sets up a symlink to point to the original plugin directory (replacing any copy if it exists), so that changes are always reflected.  
+
+### Development cycle
+1. Make changes to the plugin within the _over_react_analyzer_plugin_ directory
+1. In the _playground_ directory or in [another package you've pulled the plugin into](#pulling-in-a-local-version-of-the-plugin), restart the Analysis Server
```

Comment:
#nit from the QA is that I think it's still a little unclear that you need an IDE instance with `playground` being the content root. I think the confusion might come that people understand to open their IDE to have tools/analyzer_plugin be the content root, and then just look inside `playground` for the analyzer updates.

I think many could figure out what's going wrong, and for others it would be super easy to help figure out their issue, but I think this could be cleared up with something like the following:

```suggestion
1. In the _playground_ directory or in [another package you've pulled the plugin into](#pulling-in-a-local-version-of-the-plugin), restart the Analysis Server. Ensure that if you are using `playground` that it is the content root for your IDE instance. Otherwise, lints will not appear at all.
```

Reply:
These didn't get updated as part of this PR and need to be cleaned up anyway; are you okay addressing this in a separate PR?

Reply:
![image](https://user-images.githubusercontent.com/46691367/85345034-354c8700-b4a6-11ea-8503-2e59547c1e83.png)
![image](https://user-images.githubusercontent.com/46691367/85345049-3ed5ef00-b4a6-11ea-9e5f-59598571fcd0.png)